### PR TITLE
Outputs as CSVs

### PR DESCRIPTION
**Pull request by BenVardy**
- Remove spaces between values in csv
- Add formatting to Record output
- Fix json outputs for record data
- Update db design to fit the current db
- Remove strange formatting on csv outputs
- Remove blank newline at top of files
- Remove space at start of every line
- Add hours in times to stop excel mistaking minutes for hours
- Add ability to get full runner data

### CODE REVIEW

File: src/outputs/getEventData.js [hunk 1]
```diff
@@ -21,15 +23,24 @@ const outResults = async results => {
...
         let time = moment.duration(x.time);
-        x.time = moment({minutes: time.minutes(), seconds: time.seconds()}).format('mm:ss');
+        x.time = moment({ hours: time.hours(), minutes: time.minutes(), seconds: time.seconds()}).format('HH:mm:ss');
+        x.ageGrade = x.ageGrade + '%';
         delete x.firstEvent;
         delete x.pb;
         x.notes = x.notes.join(', ');
 
-        return x;
+        return Object.values(x).join(',');
```
Comment (ajdaniel) [inline src/outputs/getEventData.js]: `Object.values()` is not deterministic in the ordering of the values, which means you can't guarantee what order the comma separated values are. If you want to enforce an order, you will have to either use a sort function, or hard code the order in an array like `[x.event, x.pos]` etc.

**Review by markvardy — DISMISSED** [review]
I'm happy with this functionally

File: src/outputs/getEventData.js [hunk 2]
```diff
@@ -17,19 +19,37 @@ const path = `${__dirname}/../../secrets/output-data/event${eventNo}`;
...
     let out = results.map(x => {
-        delete x.event;
-        delete x.pos;
-        delete x.uuid;
+
         let time = moment.duration(x.time);
-        x.time = moment({minutes: time.minutes(), seconds: time.seconds()}).format('mm:ss');
-        delete x.firstEvent;
-        delete x.pb;
+        x.time = moment({ hours: time.hours(), minutes: time.minutes(), seconds: time.seconds()}).format('HH:mm:ss');
```
Comment (ajdaniel) [inline src/outputs/getEventData.js]: surely this can be just `x.time = time.format('HH:mm:ss');` ?

File: src/outputs/getEventData.js [hunk 3]
```diff
@@ -46,8 +66,19 @@ const outResults = async results => {
...
+    let out = [
+        'Date, #2K, #5K, #Total, #FirstTimers, #PBs',
+        [
+        dateString,
+        counts.twok,
+        counts.fivek,
+        counts.total,
+        counts.firstTimers,
+        counts.pbs
+    ].join(', ')];
```
Comment (ajdaniel) [inline src/outputs/getEventData.js:83]: 👍

**Comment by ajdaniel** [comment]
Sorry for not reviewing quicke. I have one suggestion for a code improvement but otherwise happy 👍

File: src/outputs/getEventData.js (hunk 2 above)
Comment (BenVardy) [reply src/outputs/getEventData.js]: Time is a duration and durations don't have a method `format` so it has to be changed into a moment first

**Review by markvardy — APPROVED** [review]
I'm happy with this now - please go ahead and merge.

**Pull request merged**